<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>KExportDlgDecl</class>
  <widget class="QDialog" name="KExportDlgDecl">
    <property name="windowTitle">
      <string>QIF Export</string>
    </property>
    <property name="sizeGripEnabled">
      <bool>true</bool>
    </property>
    <layout class="QVBoxLayout">
      <property name="margin">
        <number>11</number>
      </property>
      <property name="spacing">
        <number>6</number>
      </property>
      <item>
        <layout class="QHBoxLayout">
          <property name="margin">
            <number>0</number>
          </property>
          <property name="spacing">
            <number>6</number>
          </property>
          <item>
            <widget class="QLabel" name="TextLabel2">
              <property name="sizePolicy">
                <sizepolicy>
                  <hsizetype>5</hsizetype>
                  <vsizetype>7</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>From this dialog you are able to export transactions to a quicken compatible file, (known as a QIF file, because of the extension).  Please enter the path to the QIF file or select it by clicking on the Browse button.

You can choose the file's path, the account and the format of the QIF file (profile).  Choose Account to export all the transactions between the specified dates or just categories.  You can also limit the transactions that are exported by start and ending date.  Once you have pressed the Export button a message box will appear when the export has completed detailing how many transactions, categories and payees were exported.</string>
              </property>
              <property name="alignment">
                <set>Qt::AlignTop|Qt::AlignLeft</set>
              </property>
              <property name="wordWrap">
                <bool>true</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <widget class="Line" name="Line2">
          <property name="frameShape">
            <enum>QFrame::HLine</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Raised</enum>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <property name="margin">
            <number>0</number>
          </property>
          <property name="spacing">
            <number>6</number>
          </property>
          <item>
            <widget class="QLabel" name="lblFileExport">
              <property name="sizePolicy">
                <sizepolicy>
                  <hsizetype>0</hsizetype>
                  <vsizetype>1</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>File to export to:</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="KLineEdit" name="m_qlineeditFile"/>
          </item>
          <item>
            <widget class="QPushButton" name="m_qbuttonBrowse">
              <property name="text">
                <string>Browse...</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <item>
            <layout class="QVBoxLayout">
              <item>
                <widget class="QLabel" name="TextLabel2_2">
                  <property name="text">
                    <string>Account to export</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="KMyMoneyAccountCombo" name="m_accountComboBox"/>
              </item>
            </layout>
          </item>
          <item>
            <layout class="QVBoxLayout">
              <property name="margin">
                <number>0</number>
              </property>
              <property name="spacing">
                <number>6</number>
              </property>
              <item>
                <widget class="QLabel" name="TextLabel3">
                  <property name="text">
                    <string>QIF Profile</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <layout class="QHBoxLayout">
                  <property name="margin">
                    <number>0</number>
                  </property>
                  <property name="spacing">
                    <number>6</number>
                  </property>
                  <item>
                    <widget class="KComboBox" name="m_profileComboBox">
                      <property name="sizePolicy">
                        <sizepolicy>
                          <hsizetype>3</hsizetype>
                          <vsizetype>0</vsizetype>
                          <horstretch>0</horstretch>
                          <verstretch>0</verstretch>
                        </sizepolicy>
                      </property>
                      <property name="minimumSize">
                        <size>
                          <width>200</width>
                          <height>0</height>
                        </size>
                      </property>
                      <property name="duplicatesEnabled">
                        <bool>false</bool>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QPushButton" name="m_profileEditorButton">
                      <property name="text">
                        <string comment="@action create new export profile">New...</string>
                      </property>
                    </widget>
                  </item>
                </layout>
              </item>
            </layout>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <item>
            <widget class="QGroupBox" name="m_qgroupboxContents">
              <property name="title">
                <string>Contents to Export</string>
              </property>
              <layout class="QVBoxLayout">
                <property name="margin">
                  <number>11</number>
                </property>
                <property name="spacing">
                  <number>6</number>
                </property>
                <item>
                  <widget class="QCheckBox" name="m_qcheckboxAccount">
                    <property name="text">
                      <string>Account</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QCheckBox" name="m_qcheckboxCategories">
                    <property name="text">
                      <string>Categories</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </widget>
          </item>
          <item>
            <widget class="QGroupBox" name="m_qgroupboxDates">
              <property name="title">
                <string>Date Range</string>
              </property>
              <layout class="QVBoxLayout">
                <property name="margin">
                  <number>11</number>
                </property>
                <property name="spacing">
                  <number>6</number>
                </property>
                <item>
                  <layout class="QGridLayout">
                    <item row="0" column="1">
                      <widget class="kMyMoneyDateInput" name="m_kmymoneydateStart"/>
                    </item>
                    <item row="1" column="0">
                      <widget class="QLabel" name="lblEndDate">
                        <property name="sizePolicy">
                          <sizepolicy>
                            <hsizetype>0</hsizetype>
                            <vsizetype>1</vsizetype>
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                          </sizepolicy>
                        </property>
                        <property name="minimumSize">
                          <size>
                            <width>50</width>
                            <height>0</height>
                          </size>
                        </property>
                        <property name="text">
                          <string>End on:</string>
                        </property>
                        <property name="wordWrap">
                          <bool>false</bool>
                        </property>
                      </widget>
                    </item>
                    <item row="0" column="0">
                      <widget class="QLabel" name="lblStartDate">
                        <property name="sizePolicy">
                          <sizepolicy>
                            <hsizetype>0</hsizetype>
                            <vsizetype>1</vsizetype>
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                          </sizepolicy>
                        </property>
                        <property name="minimumSize">
                          <size>
                            <width>50</width>
                            <height>0</height>
                          </size>
                        </property>
                        <property name="text">
                          <string>Start on:</string>
                        </property>
                        <property name="wordWrap">
                          <bool>false</bool>
                        </property>
                      </widget>
                    </item>
                    <item row="1" column="1">
                      <widget class="kMyMoneyDateInput" name="m_kmymoneydateEnd">
                        <property name="sizePolicy">
                          <sizepolicy>
                            <hsizetype>3</hsizetype>
                            <vsizetype>1</vsizetype>
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                          </sizepolicy>
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
              </layout>
            </widget>
          </item>
          <item>
            <spacer name="Spacer7">
              <property name="sizeHint">
                <size>
                  <width>230</width>
                  <height>16</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
      <item>
        <widget class="Line" name="Line2_2">
          <property name="frameShape">
            <enum>QFrame::HLine</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <property name="margin">
            <number>0</number>
          </property>
          <property name="spacing">
            <number>6</number>
          </property>
          <item>
            <spacer name="Spacer5">
              <property name="sizeHint">
                <size>
                  <width>451</width>
                  <height>0</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="m_qbuttonOk">
              <property name="text">
                <string>Export</string>
              </property>
              <property name="autoDefault">
                <bool>true</bool>
              </property>
              <property name="default">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="m_qbuttonCancel">
              <property name="text">
                <string>Cancel</string>
              </property>
              <property name="autoDefault">
                <bool>true</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <customwidgets>
    <customwidget>
      <class>kMyMoneyDateInput</class>
      <extends>QWidget</extends>
      <header location="local">../widgets/kmymoneydateinput.h</header>
      <sizehint>
        <width>90</width>
        <height>25</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
        <hordata>1</hordata>
        <verdata>1</verdata>
      </sizepolicy>
      <pixmap>image0</pixmap>
    </customwidget>
    <customwidget>
      <class>KMyMoneyAccountCombo</class>
      <extends>QWidget</extends>
      <header location="local">../widgets/kmymoneyaccountcombo.h</header>
      <sizehint>
        <width>-1</width>
        <height>25</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
        <hordata>3</hordata>
        <verdata>5</verdata>
      </sizepolicy>
      <pixmap>image1</pixmap>
    </customwidget>
    <customwidget>
      <class>KComboBox</class>
      <extends>QComboBox</extends>
      <header location="global">kcombobox.h</header>
    </customwidget>
    <customwidget>
      <class>KLineEdit</class>
      <extends>QLineEdit</extends>
      <header location="global">klineedit.h</header>
    </customwidget>
  </customwidgets>
  <images>
    <image name="image0">
      <data format="XPM.GZ" length="646">789c6dd2c10ac2300c00d07bbf2234b7229d1be245fc04c5a3201e4615f430059d0711ff5ddb2e6bb236ec90eed134cb5a19d8ef36602af5ecdbfeeac05dda0798d3abebde87e3faa374d3807fa0d633a52d38d8de6f679fe33fc776e196f53cd010188256a3600a292882096246517815ca99884606e18044a3a40d91824820924265a7923a2e8bcd05f33db1173e002913175f2a6be6d3294871a2d95fa00e8a94ee017b69d339d90df1e77c57ea072ede6758</data>
    </image>
    <image name="image1">
      <data format="XPM.GZ" length="3368">789c9595c94eeb581086f73c4544ed50ab6eecd889ad562f1802843961a6d58b632764228c616cf5bb77f9d45fe75e11165c4c703e55d55fc319f8b152bb38daafadfc587a9abbf9b8ac9523f7585be93fcf66ef7ffff3d7bf4bcb715c93df2891d7f21f4bcb5cd4cadac1ddedc003fd02ddf9af16274075ffe3b91738f33c01276aa722b0b753ec390b7c0cce107f092e238d7fa838aa578fe713b0c5bf1aabdddd8033e49f1923be052e11df31867dc77366fe1c81cdbeab1c5b3d4d30f2d351c571dd988fc196ffda731e43cf811b98a79b06d6fa5e8c11ff1458e775054ec0dd8a1b91f9731d9ca9dd0d9513d35ff79c9b9dc7c68ddcdbf7c125ecefca16cf7e9e4994987dcfb34b73d86363e41f559cc6691ff93f8c119f2a3723b5bb8931faffc91abf691cabfd0e9c42ffd21876bf9ea90bf946c6a9f37ca66cf16e0c463fb46a8cf95e0456bb3346be3ad8fa3d34463fcf153763b33b678cf853e516e6415bc688bf31567fde04a7e8cffcd13ffbf56f4afea6ef971265f157bdd218f9d7c0e8878fc0a88ffdfe6f355a51abe1b9ad9c59ff0781fd7e62afdf2a5a79abf0f63765f377857123f2f67bb0e5f3e7256bb4f24cf3ed83fbcad4530efe99e722373e5476981faf05d679ad1aa3fe0b63ad8737c0982fed19c33ea8384ff2284f3c1f283b9be746609dc79bb19e37877837403d3bca458cf3531a23de9fdfbc147fed673bb0d6f7688cfc5960cdd737c6bc87caa2affe8dc09aefc338d17e3be026f291b1fad318ecb0bf38b0f677628cf5f1f7ab4b4c8f7363d4731b58eb2763d43f02231f9f0656bd8631f44cdfe6d535c67afaf9b8d245aef43c05a7e04439c44f8d511f2b9736cf79609d0ff44b9bd719d8ea7b0763fdd89f8742ea6beabfed73e53206df05d6fbecd618e7f311dc84ff0bd82993df7f453fd81f02abdecc58f737a56087fbf3d518f14f81f5bed17afb7dac07b58dd19fbfcfcab418383d3f73e550df36d8fa7f0eacfa57ca03ccbb3b67928ffbc6435eaf10ef92fb3ce06b1e7ef18c78cc138d80bee329dff08c6ff98eeff9811ff989e7fcecdf2ffcca6ffcfed39f0bf93610ef0f5ee5355ee70d6ef3266fc9df6deec8b3c3bbbcf7497f5fb40ff8503c8eb8cb3d6fedf1b1d0099f4accd927fd219f7bef0bf1eaf1255f719d23ff3de606279c7ed26f4addeba25629b738e39c98881c891695d45fd4a7016fd075e54d431ad1980f684253bae11ecde896ee16f41fe95e6c0f9c55def4484f92ab4573afd1a6e785fa5f65264ebc72effd22b1aff446eff421d60b5a5dd09fd39af8af134b254fbe8f8836a84d958aa3ad85fab765de3dea48979b555db4237ebbb447fbf23ea0c3c5fa799b643ea2d595ba7ba2bd2b9e0f744c27dca0d32feaef5029fe056f499767744e173413cb842ee98aea5fe877a4d7a85a577aa148a6b94fb1ecb3091d5183924ffa234a659f9c4ae61b8e6cffd2896837a945d9c2fa8e650fcadea2bc72e3b69399b842a2afe41eebbb81bbfea43f7143d983a91b714a776e4cab74e89f3a6db9899bba9b4ffadf7c82fe6ff857e7f7f7f425e25b39baf3e5fffe5cfa1fdc4beba0</data>
    </image>
  </images>
</ui>
